src/eraftpb.rs

```diff
@@ -3,7 +3,7 @@
 
 // https://github.com/Manishearth/rust-clippy/issues/702
 #![allow(unknown_lints)]
-#![allow(clippy)]
+#![cfg_attr(feature = "cargo-clippy", allow(clippy::all))]
```

Comment:
This is a generated file, maybe we should not edit it?

Reply:
That would mean blocking this and all future raft-rs PRs :( It's part of #112